src/main/java/com/diplomaProject/StudyBe/User/Controller/UserController.java

```diff
@@ -29,9 +36,17 @@ public List<User> findAll(){
     }
 
     @GetMapping("/findByID")
-    public User findByID(@RequestBody Long id){
-        return userService.findById(id);
+    public User findByID(){
+        return userService.findById(1L);
```

Comment:
What is this 1L here suppose to do?
Also the url should by only the id of the request example 
`@GetMapping("/{id}")
public ResponseData getUser(@PathVariable Long id) {
   ...
}`